templates/zerver/help/restrict-editing.md

```diff
@@ -0,0 +1,51 @@
+# Restrict editing of old messages and topics
+
+If you are the administrator of a Zulip organization, you can easily change the
```

Comment:
**the administrator** makes it sound like there can only be one administrator. I would use **an administrator** if I were you.